pkg/server/helpers.go

```diff
@@ -288,26 +285,6 @@ func (c *criContainerdService) ensureImageExists(ctx context.Context, ref string
 	return &newImage, nil
 }
 
-// loadCgroup loads the cgroup associated with path if it exists and moves the current process into the cgroup. If the cgroup
-// is not created it is created and returned.
-func loadCgroup(cgroupPath string) (cgroups.Cgroup, error) {
```

Comment:
why are we moving this helper function to cri-containerd.go? getting ready to delete it when we remove standalone mode?

Reply:
We'll have to expose this function as public function if we don't move it.